.github/workflows/android-build-verify.yml

```diff
@@ -1,5 +1,8 @@
 name: ID PASS - MOSIP Resident Application
```

Comment:
Lets change the Workflow name to 
Android build verification

Reply:
Done